build-image/first-boot.bash

```diff
@@ -156,18 +156,22 @@ ln -sfn /opt/openhabian/openhabian-setup.sh /usr/local/bin/openhabian-config
 echo "$(timestamp) [openHABian] Executing 'openhabian-setup.sh unattended'... "
 if (/bin/bash /opt/openhabian/openhabian-setup.sh unattended); then
 #if (/bin/bash /opt/openhabian/openhabian-setup.sh unattended_debug); then
-  systemctl start openhab2.service
+  service openhab2 start
```

Comment:
What is the reason to change to `service` in place of `systemctl` ?
If there are reasons for it we should change the whole codebase and track it in an individual issue.

Reply:
Tricky. Actually systemctl does not (always) work in Docker but service does.
Found a different solution meanwhile though so will remove this change